sci-electronics/gtkwave/gtkwave-3.3.125.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic gnome2-utils toolchain-funcs xdg
+
+MY_P="${PN}-gtk3-${PV}"
+DESCRIPTION="Wave viewer for LXT, LXT2, VZT, GHW and standard Verilog VCD/EVCD files"
+HOMEPAGE="https://gtkwave.github.io/gtkwave/"
+SRC_URI="https://downloads.sourceforge.net/project/${PN}/${MY_P}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2+ MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE="fasttree judy lzma packed tcl wayland X"
+
+RDEPEND="
+	app-arch/bzip2
+	dev-libs/glib:2
+	sys-libs/zlib
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/pango
+	x11-libs/gtk+:3[wayland?,X?]
+	judy? ( dev-libs/judy )
+	tcl? ( dev-lang/tcl:0= dev-lang/tk:0= )
```

Comment:
As a nit, where `x? ( ... )` has `...` expanding to more than one dependency, I like to put multiple lines.